app/models/manageiq/providers/nuage/network_manager/cloud_subnet.rb

```diff
@@ -40,23 +40,6 @@ def remove_network_ports
     network_ports.each(&:destroy)
   end
 
-  def delete_cloud_subnet_queue(userid)
-    task_opts = {
-      :action => "deleting Cloud Subnet for user #{userid}",
-      :userid => userid
-    }
-    queue_opts = {
-      :class_name  => self.class.name,
-      :method_name => 'delete_cloud_subnet',
```

Comment:
The spec is failing because it is looking for a queue item with `:method_name => delete_cloud_subnet` but openstack/vpc were queueing `raw_delete_cloud_subnet`.

The way nuage is doing it here is preferred, sorry I missed the difference in reviewing your core PR.  Could you push a follow-up to https://github.com/ManageIQ/manageiq/pull/21303 to change the queued method to be `delete_cloud_subnet` ?

Reply:
Thanks for the catch, sorry I should have noticed that in the Travis build failure

Reply:
https://github.com/ManageIQ/manageiq/pull/21307